### ISSUE TITLE
error dialog from find by case id

### ISSUE THREAD

**Issue by pieper** [ImagingDataCommons/IDC-WebApp#990]
**General description**

**Steps to reproduce the behavior**

I picked the TCGA-KIRC collection and typed a number in the Find by Case ID field and clicked Go and got the screenshot below.  It was repeatable for me.

**Screenshots**
![image](https://user-images.githubusercontent.com/126077/176004131-9c6d358b-388b-450c-b50b-b390c76ee851.png)

**Your System**

Mac, chrome.

**Other Comments**
Sorry if this is a duplicate report, I didn't find anything.

**Comment by G-White-ISB** [comment]
If you provide a case id the back end cannot find then you will generate this error. Maybe we can have a more useful error message. But do you expect to find a case id of 4164 in TCGA-KIRC?. TCGA-BP-4164 is there. We have talked about partial string matching in the search box but that is not yet implemented.

**Comment by pieper** [comment]
Yes, I expected a partial string to match.  I would think that would be a priority.

The dialog message is certainly not appropriate.  Something like "Case ID not found." would be the minimal change to clarify the current behavior.
